### DISCUSSION TITLE
Internal python script

### DISCUSSION THREAD

**Discussion by ematipico** [rome/tools#1926] [General]
Hi @rome/staff 

[This recent PR](https://github.com/rome/tools/pull/1910) showed us that we inherited some internal script that requires `python`.

This, as a consequence, requires for core contributors and staff to have python installed in their machines. This *could* create some friction, even though it's not the end of the world.

I wanted to ask you what would prefer:

1. keep `python` inside our project and update the `CONTRIBUTING.md` with mention of this script and saying that developers require having python installed

2. move the script to Rust

I vote for `2.`

**Comment by xunilrj** [comment]
> I vote for 2.

Me too.

**Comment by leops** [comment]
I agree the script would probably be more accessible if written in Rust, plus since it generates Rust code it could reuse the same quote + rustfmt infrastructure from the other generation scripts

**Comment by yassere** [comment]
I also prefer option 2.

**Comment by MichaReiser** [comment]
Option 2 would be nice but I don't think it's urgent to migrate the script.

**Reply by ematipico** [reply]
It becomes a good opportunity as "good first issue" :)

**Comment by allanlealluz** [comment]
1 is simpler, but 2... I don't know
